### ISSUE TITLE
Missing header size rule in `.tiny` modal size

### ISSUE THREAD

**Issue by icruces** [Semantic-Org/Semantic-UI#6135]
It seems a copy&paste or merging error. 

In `modal.less`, the rule:
```
/* tiny */
.ui.tiny.modal > .header:not(.ui) {
  font-size: @tinyHeaderSize;
}
```
is missing. And the small one is duplicated:
```
/* mini */
.ui.small.modal > .header:not(.ui) {
  font-size: @miniHeaderSize;
}
```
...
```
/* Small */
.ui.small.modal > .header:not(.ui) {
  font-size: @smallHeaderSize;
}
```

**Comment by icruces** [comment]
I have created a pull request with a fix: #6137

**Comment by lubber-de** [comment]
This is fixed in the community fork https://fomantic-ui.com
